src/main/java/org/gridsuite/modification/server/entities/ModificationEntity.java

```diff
@@ -76,7 +76,7 @@ protected ModificationEntity(ModificationInfos modificationInfos) {
         }
         //We need to limit the precision to avoid database precision storage limit issue (postgres has a precision of 6 digits while h2 can go to 9)
         this.date = ZonedDateTime.now(ZoneOffset.UTC).truncatedTo(ChronoUnit.MICROS);
-
+        this.stashed = modificationInfos.getStashed();
```

Comment:
Move it to the assignAttributes method ?

Reply:
Maybe it's better because not updatable ?

Reply:
ok i will move it now